streaming/base/storage/upload.py

```diff
@@ -203,6 +204,26 @@ def upload_file(self, filename: str):
             )
         self.clear_local(local=local_filename)
 
+    def bucket_exists(self, remote: str):
```

Comment:
can we call this `check_bucket_exists` or `validate_bucket` or similar?

Reply:
renamed it to `check_bucket_exists` in https://github.com/mosaicml/streaming/pull/212/commits/1089fb116dd3fc1b8c797d0960eb27bcb6fb8320.